intellij-plugin-verifier/verifier-test/src/test/java/com/jetbrains/pluginverifier/tests/KotlinInternalModifierUsageTest.kt

```diff
@@ -127,10 +127,9 @@ class KotlinInternalModifierUsageTest {
     val plugin = prepareUsage(pluginSpec) { usageClassUdt }
 
     verify(ide, plugin).run {
-      assertEquals(1, size)
+      assertEquals(0, size)
```

Comment:
why not simply disable/mute the test altogether?

Reply:
Using `@Ignore` annotations instead. Restored the original behaviour.